pkg/ui/workspaces/cluster-ui/src/store/localStorage/localStorage.reducer.ts

```diff
@@ -98,6 +110,23 @@ const defaultSortSettingSchemaInsights: SortSetting = {
   columnTitle: "insights",
 };
 
+const defaultDatabasesSortSetting: SortSetting = {
+  ascending: true,
+  columnTitle: "name",
+};
+
+export const defaultDatabaseDetailsTableSortSetting: SortSetting = {
+  ascending: true,
+  columnTitle: "name",
+};
+
+export const defaultDatabaseDetailsGrantSortSetting: SortSetting = {
+  ascending: true,
+  columnTitle: "name",
+};
```

Comment:
nit: These are all the same object. I'd create this object once and then spread it below.

Reply:
Done.